validator/Cargo.toml

```diff
@@ -17,6 +17,7 @@ log = "0.4.8"
 indicatif = "0.13.0"
 reqwest = { version = "0.9.22", default-features = false }
 serde_json = "1.0.41"
+solana-cli = { path = "../cli", version = "0.21.0" }
```

Comment:
I'm not sure depending on "cli" from "validator" can be justified...

Reply:
I’d rather not have validator depend on cli.    It’s long overdue that we create a common clap-validators/ crate where we can easily share these validators between all the tools. 

 Care to add that common crate first and then do the input validator cleanup here?  We can do that in a separate PR from the check_vote_account() changes here.

Reply:
Heh, that's what I wanted to do ideally. I'll do the cleaning job now that I got the approval. :)

Reply:
Done at https://github.com/solana-labs/solana/pull/6812